src/test/java/com/julienvey/trello/integration/CardGetITCase.java

```diff
@@ -111,4 +114,17 @@ public void testGetCardBoard() {
         assertThat(cardBoard.getId()).isEqualTo("518baad5b05dbf4703004852");
     }
 
+    @Test
+    @Ignore("Not sure how to properly test the mutation of state since there is no way to revert the change currently")
```

Comment:
please take a look at CardIt Scala class. it is fine to create temporary data in that test project and just leave it there. let's have an active test for this case in CardIt itself.

Reply:
@alexeyOnGitHub I've added the test to CardIt class, please take a look.